### PR TITLE
feat: add list tags methods for scripts and tests

### PR DESCRIPTION
**Pull request by nicufk**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: client/tests/tests.go [hunk 1]
```diff
@@ -45,6 +46,23 @@ func (s TestsClient) List(namespace string, tags []string) (*testsAPI.TestList,
...
+func (s TestsClient) ListTags(namespace string) ([]string, error) {
+	tags := []string{}
+	list := &testsAPI.TestList{}
+	err := s.Client.List(context.Background(), list, &client.ListOptions{Namespace: namespace})
+	if err != nil {
+		return tags, err
+	}
+
+	for _, test := range list.Items {
+		tags = append(tags, test.Spec.Tags...)
```
Comment (vsukhin) [inline client/tests/tests.go:58]: I would add only new tags there, rather than cleaning duplicates later

File: client/scripts/scripts.go [hunk 2]
```diff
@@ -45,6 +46,23 @@ func (s ScriptsClient) List(namespace string, tags []string) (*scriptsAPI.Script
...
+func (s ScriptsClient) ListTags(namespace string) ([]string, error) {
+	tags := []string{}
+	list := &scriptsAPI.ScriptList{}
+	err := s.Client.List(context.Background(), list, &client.ListOptions{Namespace: namespace})
+	if err != nil {
+		return tags, err
+	}
+
+	for _, test := range list.Items {
+		tags = append(tags, test.Spec.Tags...)
```
Comment (vsukhin) [inline client/scripts/scripts.go:58]: I would add only new tags there, rather than cleaning duplicates later

**Review by vsukhin — APPROVED** [review]
looks good, one minor comment

File: client/tests/tests.go (hunk 1 above)
Comment (jasmingacic) [reply client/tests/tests.go:58]:
```
	var (
		a intsets.Sparse
		b bool
	)
	b = a.Insert(9)
	println(b) // true
	b = a.Insert(9)
	println(b) // false
```

Comment (jasmingacic) [reply client/tests/tests.go:58]: or simpler 
```
func AppendIfMissing(slice []int, i int) []int {
    for _, ele := range slice {
        if ele == i {
            return slice
        }
    }
    return append(slice, i)
}
```

Comment (nicufk) [reply client/tests/tests.go:58]: what is your concern here? I can do it but the code will get more complicated and performance wise there is no difference.

Comment (jasmingacic) [reply client/tests/tests.go:58]: actually you are right
No need to overdo it

**Review by jasmingacic — APPROVED** [review]

**Pull request merged**